### PR TITLE
Add a collection of DevOps related projects

### PR DESCRIPTION
**Pull request by AbdelouahabMbarki**
fix: #50 
description:
Adding a collection of kubesphere DevOps related projects to doc

### CODE REVIEW

File: docs/projects.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+| [s2i-binary-container](https://github.com/kubesphere/s2i-binary-container) |  Binary S2I Builder image  |
+| [s2i-golang-container](https://github.com/kubesphere/s2i-golang-container) | Golang S2I Builder image  |
+| [kubesphere](https://github.com/kubesphere/kubesphere)| The container platform tailored for Kubernetes multi-cloud, datacenter, and edge management ⎈ 🖥  ☁️  |
+| [ks-jenkins](https://github.com/kubesphere/ks-jenkins)  | Jenkins distribution for Kubesphere  |
+| [docker-agent](https://github.com/kubesphere/docker-agent)  | Base Docker image for Jenkins Agents  |
+| [docker-inbound-agent](https://github.com/kubesphere/docker-inbound-agent)  | Docker image for a Jenkins agent which can connect to Jenkins using TCP or Websocket protocols  |
+| [jenkins-library](https://github.com/kubesphere/jenkins-library)  | KubeSphere Jenkins Pipeline Library  |
+| [jenkins-docker](https://github.com/kubesphere/jenkins-docker)  | Docker official jenkins repo |
+| [kubernetes-deploy-plugin](https://github.com/kubesphere/kubernetes-deploy-plugin) | jenkins kubernetes deploy plugin  |
```
Comment (LinuxSuRen) [inline docs/projects.md:19]: we don't need these projects. Most of them are just a fork project.

File: docs/projects.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+| [s2irun](https://github.com/kubesphere/s2irun) | Run s2i in pod |
+| [s2ioperator](https://github.com/kubesphere/s2ioperator) | Operator for Source to image  |
+| [s2i-python-container](https://github.com/kubesphere/s2i-python-container) | Python S2I Builder image |
+| [s2i-java-container](https://github.com/kubesphere/s2i-java-container) | Java S2I Builder image  |
+| [s2i-base-container](https://github.com/kubesphere/s2i-base-container) | Dockerfiles which serve as base images for s2i  |
+| [s2i-nodejs-container](https://github.com/kubesphere/s2i-nodejs-container) | NodeJS  S2I Builder image  |
+| [s2i-java-runtimeImage](https://github.com/kubesphere/s2i-java-runtimeImage) | A java runtime image for s2i |
+| [s2i-binary-container](https://github.com/kubesphere/s2i-binary-container) |  Binary S2I Builder image  |
+| [s2i-golang-container](https://github.com/kubesphere/s2i-golang-container) | Golang S2I Builder image  |
+| [kubesphere](https://github.com/kubesphere/kubesphere)| The container platform tailored for Kubernetes multi-cloud, datacenter, and edge management ⎈ 🖥  ☁️  |
```
Comment (LinuxSuRen) [inline docs/projects.md:13]: This project is not part of DevOps.

**Comment by LinuxSuRen** [comment]
Any updates?

**Review by LinuxSuRen — APPROVED** [review]

**Comment by AbdelouahabMbarki** [comment]
thank you @LinuxSuRen  i submitted the updates

**Pull request merged**